lib/modules/dosomething/dosomething_kudos/includes/Kudos.php

```diff
@@ -86,6 +93,16 @@ public static function find(array $filters = []) {
   }
 
   /**
+   * Get the approved reactions.
+   * @return array
+   */
+  public static function getApprovedReactions() {
+    $kudos = new static;
```

Comment:
Oh, true. I wasn't sure if it's normally a best practice to be retrieving property values on a class using getter methods, rather than grabbing a static property directly... Maybe that's not actually a thing? haha.

I can update!